### PR TITLE
Refactors copy resource link success message.

### PR DESCRIPTION
**Pull request by boyan-tonchev**
## What
Changes implementation of notification when an url of result link is copied into clipboard.

## Why
There is a new implementation of event fired by "ontotext-yasgui" when an url link is copied.
## How
Updates the version of "ontotext-yasgui". Register a notification event handler in "ontotext-yasgui" output event emitter.
Additional work:
Added models for:
- EventData - event data model for all events fired by "ontotext-yasgui";
- EventDataType - describes all possible events which can be fired by "ontotext-yasgui";
- NotificationMessageType - describes all possible notification message types: "success", "warning", "error";
- YasrPluginName - describes the names of yasr plugins.

### CODE REVIEW

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

File: src/models/ontotext-yasgui/event-data-type.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Holds all possible values of {@see EventData} types.
+ */
+export class EventDataType {
```
Comment (svilenvelikov) [inline src/models/ontotext-yasgui/event-data-type.js:4]: These can be just simple objects like 
```javascript
export const EventDataType = {
    'DOWNLOAD_AS': 'downloadAs',
    'NOTIFICATION_MESSAGE': 'notificationMessage'
};
```

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**